Dockerfile

```diff
@@ -5,8 +5,8 @@ RUN apt-get update && apt-get install -y openjdk-7-jre curl
 RUN mkdir /app
 WORKDIR /app
 
-RUN curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.14.2/maxwell-0.14.2.tar.gz | tar zxvf -
-RUN cp -r maxwell-*/* . && rm -rf maxwell-*
+RUN curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.13.1/maxwell-0.13.1.tar.gz \
```

Comment:
Is there any particular reason the release tag is being changed here from 0.14.2 to 0.13.1?